CircleciScripts/carthage-build.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+################################################################################
+# Workaround for https://github.com/Carthage/Carthage/issues/3019
+# from https://github.com/Carthage/Carthage/issues/3019#issuecomment-665136323
+#
+# Update this to include new values as new Xcode 12 versions are released
+################################################################################
+
+# carthage-build.sh
+# Usage example: ./carthage-build.sh --platform iOS
+
+set -euo pipefail
+
+xcconfig=$(mktemp /tmp/static.xcconfig.XXXXXX)
+trap 'rm -f "$xcconfig"' INT TERM HUP EXIT
```

Comment:
I've used `trap` with `INT` and `HUP` but not `EXIT` -- I am assuming that that when the script runs to completion, that we will remove the file?

Reply:
Yep, however, there's actually a bug--thanks for pointing that out! If I execute defensively (like I you pointed out above) then the trap gets assigned to the current shell, and I don't know when or if it would execute. Rather than executing with the current shell, I'll change the `carthage-build` execution to directly invoke `bash`.